LibreNMS/Device/YamlDiscovery.php

```diff
@@ -127,6 +118,51 @@ public static function discover(OS $os, $class, $yaml_data)
         return $items;
     }
 
+    /**
+     * @param Device $device Device we are working on
+     * @param array $data Array derived from YAML
+     * @return string
+     */
+    public static function computeNumericalOID($device, $data)
+    {
+        static $num_oid_cache;
+        static $num_oid_cache_negative;
+        // Compute md5 of the query
+        $md5 = md5($data['value'] . $data['oid'] . $device['dynamic_discovery']['mib']);
```

Comment:
Why md5? You have your key by just concating them together

Reply:
"keep it simple", true. This is corrected as well.